src/plugins/api-proxy/handler.js

```diff
@@ -0,0 +1,35 @@
+import Joi from 'joi';
```

Comment:
This file was moved from src/apiProxy/apiProxyHandler.js, but `parseRequestQueries` was moved into it from `src/util/apiUtils.js` because we only need to parse the request queries in the api proxy endpoint (handled here), not anywhere else